docs/UserGuide.md

```diff
@@ -275,10 +279,12 @@ Here’s a reference table of available flags and the type of data they correspo
 
 ## 4.4 Arguments
 
-Arguments are the values that follow each flag in a command. **Arguments cannot be empty**, and each must meet specific parsing and format requirements to ensure proper execution of the command.
+Arguments are the values that are used in a command. **Arguments cannot be empty** and each must meet specific parsing and format requirements to ensure proper execution of the command.
```

Comment:
Hmm, I personally prefer `follow each flag` since it suggests it comes right after flags. What does the rest of the team think?

Reply:
For this, I kept the initial phrasing for the arguments specifically for Flags (New section 4.1), so this amendment is for the general Arguments

Currently I split into Flag Arguments and Non-Flag arguments for clarity, albeit the only non-flag argument we currently have is Index, so I'm not too sure if this is the most appropriate. What do the rest think?